### ISSUE TITLE
List of tags for provider fails WCAG contrast checks

### ISSUE THREAD

**Issue by radioprotector** [radremedy/radremedy#241]
To pass, I think it needs to be at least `#716e6e` if not darker (such as `#525050`).
